### PR TITLE
Including README and examples

### PR DESCRIPTION
**Pull request by abramvandergeest**

### CODE REVIEW

File: examples/golangWithTrigger/main.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	// "github.com/project-flogo/contrib/trigger/timer"
+	"github.com/skothari-tibco/csvtimer"
+
+	cml "github.com/project-flogo/cml/action"
```
Comment (skothari-tibco) [inline examples/golangWithTrigger/main.go]: This should be updated.

File: examples/golangWithTrigger/go.mod [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	github.com/project-flogo/contrib/activity/syncaction v0.0.0
+	github.com/project-flogo/core v0.9.4-0.20190829220729-31eb91f2b8a7
+	github.com/project-flogo/legacybridge v0.9.1
+	github.com/project-flogo/ml v0.1.2
+	github.com/project-flogo/stream v0.2.0
+	github.com/project-flogo/stream/activity/aggregate v0.2.0
+	github.com/skothari-tibco/csvtimer v0.9.1
+)
+
+replace github.com/project-flogo/cml/action => /Users/avanderg@tibco.com/working/go_working/cml/action
```
Comment (skothari-tibco) [inline examples/golangWithTrigger/go.mod]: This should be updated too.

File: examples/golangWithTrigger/main.go (hunk 1 above)
Comment (abramvandergeest) [reply examples/golangWithTrigger/main.go]: I have done this for all the examples in the below commit.

File: examples/golangWithTrigger/go.mod (hunk 2 above)
Comment (abramvandergeest) [reply examples/golangWithTrigger/go.mod]: I have done this for all the examples in the below commit.

**Review by skothari-tibco — APPROVED** [review]

**Review by mellistibco — APPROVED** [review]

**Pull request merged**